src/server/routes/status.rs

```diff
@@ -0,0 +1,44 @@
+use actix_web::{
+    web::Data,
+    HttpResponse,
+};
+use diesel::{QueryDsl, RunQueryDsl};
+use diesel::dsl::count;
+use serde::Serialize;
+
+use crate::database::schema::transactions::{self, id};
+use crate::server::RuntimeContext;
+use crate::{server::error::ValidatorServerError, key_manager};
+use crate::server::routes::sign::Config;
+
+#[derive(Serialize)]
+struct StatusBody {
+    total_txs: i64,
+    epoch: u128,
+    next_epoch: u128,
+    previous_epoch: u128
+}
+
+pub async fn index<Context, KeyManager>(
```

Comment:
Maybe this should be renamed as `status`.

Reply:
Yes, mb